Assets/LeapMotionModules/UI/Scripts/MinimalBody.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Leap.Unity;
+using UnityEngine;
+
+[ExecuteBefore(typeof(SpringBase))]
```

Comment:
We should make a note somewhere to fix the execution order attributes, since they can bug out sometimes still.